epochStart/bootstrap/process.go

```diff
@@ -1210,22 +1210,23 @@ func (e *epochStartBootstrap) createResolversContainer() error {
 	//  this one should only be used before determining the correct shard where the node should reside
 	log.Debug("epochStartBootstrap.createRequestHandler", "shard", e.shardCoordinator.SelfId())
 	resolversContainerArgs := resolverscontainer.FactoryArgs{
-		ShardCoordinator:                e.shardCoordinator,
-		MainMessenger:                   e.mainMessenger,
-		FullArchiveMessenger:            e.fullArchiveMessenger,
-		Store:                           storageService,
-		Marshalizer:                     e.coreComponentsHolder.InternalMarshalizer(),
-		DataPools:                       e.dataPool,
-		Uint64ByteSliceConverter:        uint64ByteSlice.NewBigEndianConverter(),
-		NumConcurrentResolvingJobs:      10,
-		DataPacker:                      dataPacker,
-		TriesContainer:                  e.trieContainer,
-		SizeCheckDelta:                  0,
-		InputAntifloodHandler:           disabled.NewAntiFloodHandler(),
-		OutputAntifloodHandler:          disabled.NewAntiFloodHandler(),
-		MainPreferredPeersHolder:        disabled.NewPreferredPeersHolder(),
-		FullArchivePreferredPeersHolder: disabled.NewPreferredPeersHolder(),
-		PayloadValidator:                payloadValidator,
+		ShardCoordinator:                    e.shardCoordinator,
+		MainMessenger:                       e.mainMessenger,
+		FullArchiveMessenger:                e.fullArchiveMessenger,
+		Store:                               storageService,
+		Marshalizer:                         e.coreComponentsHolder.InternalMarshalizer(),
+		DataPools:                           e.dataPool,
+		Uint64ByteSliceConverter:            uint64ByteSlice.NewBigEndianConverter(),
+		NumConcurrentResolvingJobs:          10,
```

Comment:
we should create a task to remove the magic numbers.

Reply:
done: MX-15149